src/pages/ReportDetailsPage.tsx

```diff
@@ -246,7 +246,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
 
     const shouldShowLeaveButton = ReportUtils.canLeaveChat(report, policy);
 
-    const reportName = ReportUtils.isDeprecatedGroupDM(report) || isGroupChat ? ReportUtils.getGroupChatName(undefined, false, report) : ReportUtils.getReportName(report);
+    const reportName = ReportUtils.getReportName(report);
```

Comment:
Do we need to remove `isDeprecatedGroupDM` here? I see `getReportName` doesn't have a clear condition for `isDeprecatedGroupDM`

Reply:
I don't know why we don't check for `isDeprecatedGroupDM` in `getReportName`, but the other usages of `getReportName` doesn't check for `isDeprecatedGroupDM` too.

Reply:
I think `getReportName` may cover it by [the default return](https://github.com/Expensify/App/blob/578006fe4d2b7ba13155e795bcef541019aad8ea/src/libs/ReportUtils.ts#L3776-L3792), but I am not sure about that. Do we have any test steps can help us to test this PR on deprecated group?

Reply:
Yeah, I notice the default return too and it actually apply the limit by default.
https://github.com/Expensify/App/blob/8c69c58dfa7f6a9ab079e560ff026c1aed12be8e/src/libs/ReportUtils.ts#L3774-L3781

>  Do we have any test steps can help us to test this PR on deprecated group?

I don't think there is a way to create a deprecated group anymore. The only way to test it by opening the old existing deprecated group DM.

I have one here and using the logic from `main`, the LHN title will show the participants with limit it to 5 participants, while the details page shows Chat Report as the name.
https://github.com/user-attachments/assets/a2c25382-b301-44da-9c2e-b7ec614d704e

If we change it using this PR, the details title will show all the participants with a limit too.

https://github.com/user-attachments/assets/e6ea111a-2ad0-4b8a-9a35-9783ec5a4338

Reply:
I am able to reproduce it now. But why LHN title not the same as header and details titles?

Reply:
Note, the above issue ^ not have been happened on native android and iOS

Reply:
I haven't investigated it yet, but I notice it happens too without this PR.

Reply:
I think we need to investigate it

Reply:
Thanks for the investigations, but that fix still not work completely, let's keep it to fix separately here https://github.com/Expensify/App/issues/47803. I just wanted to confirm it is not a regression from our PR

Reply:
I've reverted the commit. Lmk if I misunderstood it.